sdk/healthinsights/Azure.Health.Insights.RadiologyInsights/src/Generated/Docs/RadiologyInsightsClient.xml

```diff
@@ -74,7 +74,7 @@ RadiologyInsightsData radiologyInsightsData = new RadiologyInsightsData(new Pati
             SpecialtyType = SpecialtyType.Pathology,
             AdministrativeMetadata = new DocumentAdministrativeMetadata
             {
-                OrderedProcedures = {new FhirR4Extendible
```

Comment:
This change should be fine:
- It's still beta
- next version is coming
- It's a good change. Previously we only generate `FHirR4Extendible` class for definition like `OrderedProcedures is FHirR4Extendible`. I think the current generated codes are the correct results

Reply:
I am good with this change. Could you confirm with service team?
cc @m-nash for awareness.

Reply:
Java team has confirmed with service team. We should be good to go.